crates/dbsp/src/trace/spine_async/mod.rs

```diff
@@ -197,6 +203,9 @@ where
         debug_assert!(!batch.is_empty());
         let level = Spine::<B>::size_to_level(batch.len());
         self.slots[level].loose_batches.push_back(batch);
+
+        gauge!(BATCHES_PER_LEVEL, "worker" => Runtime::worker_index_str(), "level" => LEVELS_AS_STR[level], "id" => self.ident)
+            .set(self.slots[level].loose_batches.len() as f64);
```

Comment:
I would write:

```rust
            .set(self.slots[level].n_batches() as f64);
```

instead, because batches being merged are still part of the level.

Reply:
Oh, I see you defined BATCHES_PER_LEVEL as loose batches only. Why?